src/avatars.scss

```diff
@@ -0,0 +1,54 @@
+@mixin avatar-base {
+  border-radius: 50%;
+  display: inline-block;
+  object-fit: cover;
+  overflow: hidden;
+  position: relative;
+
+  &:after {
+    align-items: center;
+    background: linear-gradient(var(--black-tint-70), var(--black-tint-40));
+    color: var(--white);
+    content: attr(alt);
+    display: inline-flex;
+    height: 100%;
+    justify-content: center;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+}
+
+.avatar-large {
+  @include avatar-base;
+  font-size: 1rem;
+  height: 48px;
+  width: 48px;
+}
+
+.avatar-medium {
+  @include avatar-base;
+  font-size: 0.75rem;
+  height: 32px;
+  width: 32px;
+}
+
+.avatar-small {
+  @include avatar-base;
+  font-size: 0.75rem;
+  height: 24px;
+  width: 24px;
+}
+
+// Example .avatar-overlap rules:
+// .avatar-overlap > img:nth-child(2) { transform: translateX(-25%); z-index: -1; }
+// .avatar-overlap > img:nth-child(3) { transform: translateX(-50%); z-index: -2; }
+.avatar-overlap {
+  @for $i from 1 through 19 {
```

Comment:
Hmm, not sure I like this 19 here. I assume we want to target all children but the first? Would `:not(:first-child)` work?

Ah, I see you're also using the index to calculate the transform property and the z-index. Not sure about the z-index, but `margin-left: -1rem` could work instead of transforming, though you might need to adjust the value for different sizes. I think that's still better than having a separate class for each child that only goes up to 19.

Reply:
So the `-1rem` solution would work, but would be different for each size of avatar. I figured, since I needed to set the `z-index` anyway, I might as well loop it all into one ruleset.

Any thoughts on a better solution for `z-index`?

Reply:
Not really, I would ask Cody if it's really that bad if they stack the opposite way 🙃

Reply:
Ay! Design don't care, so I'll go with the simpler size-specific approach